README.md

```diff
@@ -22,16 +22,20 @@
 
 ### **The Open-source Developer-first LLMOps Platform**
 
-
 Building production-ready LLM-powered applications is currently very difficult. It involves countless iterations of prompt engineering, parameter tuning, and architectures.
 
 Agenta provides you with the tools to quickly 🔄 **iterate**, 🧪 **experiment**, and ⚖️ **evaluate** your LLM apps. All without imposing any restrictions on your choice of framework, library, or model.
 
 https://github.com/Agenta-AI/agenta/assets/57623556/99733147-2b78-4b95-852f-67475e4ce9ed
 
+## News
+
```

Comment:
+ Let's move this to our docs
+ Let's rename this section "Migrations" and add a link to the docs
+ Let's add a command on how to run the Python script

+ add it to our docs and a small link